asip_v2/archive.py

```diff
@@ -281,29 +281,30 @@ def read_icechart_coding(self, fil, filename):
 
     @staticmethod
     def get_the_mask_of_sar_size_data(sar_names, fil, distance_threshold):
+        mask_sar_size = False
         for str_ in sar_names+['polygon_icechart']:
             mask = np.ma.getmaskarray(fil[str_][:])
-            mask = np.ma.mask_or(mask, mask)
+            mask_sar_size = np.ma.mask_or(mask_sar_size, mask, shrink=False)
             # not only mask itself, but also being finite is import for the data. Thus, another
             # mask also should consider and apply with 'mask_or' of numpy
             mask_isfinite = ~np.isfinite(fil[str_])
-            mask = np.ma.mask_or(mask, mask_isfinite)
+            mask_sar_size = np.ma.mask_or(mask_sar_size, mask_isfinite, shrink=False)
         # ground data is also masked
-        mask_sar_size = np.ma.mask_or(mask, np.ma.getdata(
-            fil['distance_map']) <= distance_threshold)
+        mask_sar_size = np.ma.mask_or(mask_sar_size, np.ma.getdata(fil['distance_map']) <=
+                                                                                 distance_threshold)
         return mask_sar_size
 
-    @staticmethod
```

Comment:
Is it not static anymore? Why get_the_mask_of_sar_size_data is still static?

Reply:
because of self.ASPECT_RATIO is it no more static. I have hardcoded the value of 50 for it in the previous code which I correct it with this PR.